docs/developers/guard-accounts.md

```diff
@@ -150,11 +150,25 @@ There is no solution released for the **xPortal Wallet**. As soon as there is on
 
 ### Ledger
 
-There is no solution released for the **Ledger**. As soon as there is one, it will be documented and referenced here.
+There is no official solution released for the **Ledger**. As soon as there is one, it will be documented and referenced here.
 
-## For integrators
+:::note
+In the meantime you can use the **MultiversX Guardians Test - BETA** version:
```

Comment:
you can also state that this application is not yet approved and/or reviewed by Ledger and should only be used for testing, such as The great heist. It is not recommended for production use